sound/soc/intel/boards/sof_sdw_max98373.c

```diff
@@ -0,0 +1,79 @@
+// SPDX-License-Identifier: GPL-2.0
```

Comment:
GPL-v2.0-only

Reply:
Will change to 'GPL-2.0-or-later' Thanks.